src/authentication/footer/styles.scss

```diff
@@ -0,0 +1,12 @@
+@import '../styles';
+
+.invite-footer {
+  @include base();
```

Comment:
(doubt) Why do we need to incluse "base" in the footer file? Instead shouldn't we include "footer as base()" in the main `styles.scss` file?

Reply:
yeah we can do that for sure. Updated